tests/python_functional/functional_tests/logpath/test_flags_fallback.py

```diff
@@ -54,20 +48,23 @@ def test_flags_fallback(tc):
 
     config.create_logpath(statements=[file_source, inner_logpath1, inner_logpath2])
 
-    bsd_message1 = write_message_with_fields(tc, file_source, hostname="host-A", message="message from host-A")
-    bsd_message2 = write_message_with_fields(tc, file_source, hostname="host-B", message="message from host-B")
+    log_message1 = LogMessage().hostname("host-A")
```

Comment:
Abstracting out this chunk into `write_message_wit_fields` seemed nice to me. Is there a reason that this cannot work anymore?

Same question for the other testcases.

Reply:
Thanks for the note. I will come up with sg here.